src/awfy.js

```diff
@@ -951,6 +959,8 @@ const DESKTOP_APPS = {
   },
 };
 
+const DESKTOP_LIVE_SITES = { ...DESKTOP_SITES, ...LIVE_SITES };
```

Comment:
The `LIVE_SITES` is an additional list of site that we run on mobile. I'm not sure if we're planning to also run these on desktop (I'll ask on the [live sites patch](https://phabricator.services.mozilla.com/D100780)). That said, with #358 we can probably move towards a single list of sites as we'll only show results when they're available.